autospec/tarball.py

```diff
@@ -248,12 +298,10 @@ def download_tarball(target_dir):
 
     build.download_path = target
     call("mkdir -p {}".format(build.download_path))
-
-    # locate the tarball locally or download
-    return check_or_get_file(url, tarfile)
+    return target
 
 
-def convert_version(ver_str):
+def convert_version(ver_str, name):
```

Comment:
The new `name` parameter here is unused in the function. Is it supposed to be used, or is it left over from a previous iteration of the PR?

Reply:
It is being used, see line 323. This is a weird change though because I'm masking a global so you don't see any source change in the function. Would you prefer I used a different variable name and update the source to use that?

Reply:
No, sorry, I missed the variable references within the function. My comment was invalid.